src/main/java/bio/terra/janitor/db/JanitorDao.java

```diff
@@ -29,9 +30,13 @@ public JanitorDao(JanitorJdbcConfiguration jdbcConfiguration) {
     jdbcTemplate = new NamedParameterJdbcTemplate(jdbcConfiguration.getDataSource());
   }
 
-  /** Creates the tracked_resource record and adding labels. */
+  /**
+   * Creates the tracked_resource record and adding labels.
+   *
+   * <p>Note that we assume int input {@code cloudResourceUid} is valid.
+   */
   @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
-  public UUID createResource(
+  public String createResource(
```

Comment:
I think this should return the TrackedResourceId. The fact that we have an id for the tracked resource is not specific to teh JanitorDao, e.g. we want to support gets with it and will do business logic per id. It's useful to have a type for it and not type it as a UUID or a String.

Reply:
Done. 
I think we will need a class to represents most of the db entities.  Adding a class for TrackedResource was planed as part of the get endpoint ticket.